ksql-cli/src/main/java/io/confluent/ksql/cli/Cli.java

```diff
@@ -216,7 +216,9 @@ void handleLine(final String line) throws Exception {
       return;
     }
 
-    handleStatements(line);
+    if (!terminal.maybeHandleCliSpecificCommands(trimmedLine)) {
```

Comment:
Why have you moved this from in `Console`?

I don't see what it's gained you.  You now need to call back into the console, (both here and all tests), rather than just having the console internally handle cli specific commands...

Reply:
See my response below - maybe I'm the only one who has been confused by this, but every time I look at this code I get confused.